File: tests/fixtures/zig/basic.zig
// expect: total=16 code=8 comment=4 blank=4 shebang=0
// Simple Zig fixture.

const std = @import("std");

/// Doc comment above a type.
const Thing = struct {
	//! Top-level comment inside a container.
	const A: f64 = 2.93821;
};

pub fn main() {
	const a = Thing.A;
	std.debug.print("{d}", a);
}

File: crates/cli/tests/fixtures/rust/basic.rs
// expect: total=12 code=7 comment=3 blank=2 shebang=0
// simple rust fixture

fn greet(name: &str) -> String {
	format!("Hello, {name}!")
}

fn main() {
	// print the greeting
	let msg = greet("world");
	println!("{msg}");
}
